airbyte-ci/connectors/pipelines/tests/test_actions/test_environments.py

```diff
@@ -3,41 +3,40 @@
 #
 
 import pytest
-from connector_ops.utils import Connector
 from pipelines.actions import environments
-from pipelines.contexts import PipelineContext
+from pipelines.contexts import ConnectorContext
 
 pytestmark = [
     pytest.mark.anyio,
 ]
 
 
 @pytest.fixture
-def python_connectors() -> Connector:
-    return [
-        Connector("source-openweather"),  # setup.py based
-        Connector("destination-duckdb"),  # pyproject.toml based
```

Comment:
I think we lost this test case for pyproject.toml

Reply:
It's back in an upstream stack: https://github.com/airbytehq/airbyte/pull/30474/files#diff-8c52a9f4479f21f181275cfec50b1ddce0c0302cfd6e12af1c5404e206158ea7R22